src/nodes/corenodes/input/noise_image_node/noise_image_node.py

```diff
@@ -50,27 +51,43 @@ def NodeDndEventHook(self):
         self.RefreshNodeGraph()
 
     def NodeInitProps(self):
-        noise_seed = api.PositiveIntegerProp(
-            idname="noise_seed",
-            default=1,
-            min_val=0,
-            max_val=100,
-            fpb_label="Noise Seed"
+        height = api.IntegerProp(
+            idname="height",
+            default=512,
+            min_val=1,
+            max_val=10000,
+            show_p=True,
+            fpb_label="Height"
         )
 
-        self.NodeAddProp(noise_seed)
+        width = api.IntegerProp(
+            idname="width",
+            default=512,
+            min_val=1,
+            max_val=10000,
+            show_p=True,
+            fpb_label="Width"
+        )
+        self.NodeAddProp(height)
+        self.NodeAddProp(width)
 
-    def NodeEvaluation(self, eval_info):
-        noise_seed = self.EvalProperty(eval_info, "noise_seed")
+    def NodeInitOutputs(self):
+        self.outputs = {
+            "image": api.Output(idname="image", datatype="IMAGE", label="Image"),
+        }
 
+    def NodeEvaluation(self, eval_info):
         render_image = api.Image()
+        props = {}
+        shader_src = "nodes/corenodes/input/noise_image_node/noise_image_node.glsl"
+        image = Image((eval_info.node.properties['width'].value, eval_info.node.properties['height'].value))
```

Comment:
Style: It is preferable to make a variable declaration for each property. 
Code: Always use the ``EvalProperty`` method to get the property values, otherwise you may be accessing a non-finalized value.

```python
image_width = self.EvalProperty(eval_info, "width") 
image_height =  self.EvalProperty(eval_info, "height") 
...
image = Image((image_width, image_height))
```